k8s/nginx/staging/global-config.yaml

```diff
@@ -29,12 +29,13 @@ data:
       if ($request_uri ~ '(/api/v1/users/emailLogin)|(/api/v2/users/emailLogin)') { return 200; }
       if ($request_uri ~ '(/api/v1/users/forgotPassword)|(/api/v2/users/forgotPassword)') { return 200; }
       if ($request_uri ~ '(/api/v1/users/candidates/register)|(/api/v2/users/candidates/register)') { return 200; }
-      if ($request_uri ~ '(/api/v1/users/verify)|(/api/v1/users/verify)') { return 200; }
+      if ($request_uri ~ '(/api/v1/users/verify)|(/api/v2/users/verify)') { return 200; }
       if ($request_uri ~ '/airflow/api/v') { return 200; }    
       if ($request_uri ~ '/grafana/api') { return 200; }    
-      if ($request_uri ~ '/api/v1') { return 200; } # API v1 endpoint requests
       if ($request_uri !~ '/api/v') { return 200; } # None API requests 
```

Comment:
Thanks @123MwanjeMike 

Observation that is somehow unrelated to the PR....
what is this` /airflow/api/v` I am seeing in the configs? Oversight/ incomplete/intentional?

Reply:
Thanks @123MwanjeMike